### ISSUE TITLE
Use decimal types instead of float

### ISSUE THREAD

**Issue by WhyAsh5114** [WhyAsh5114/MyFit#120]
## Feature request
Currently using float types in Prisma schema, but Decimal seems to be better. Might or might not do this

**Comment by Schroedingberg** [comment]
I'd say YAGNI in this domain. Most equipment has a precision of about 1.25 kg, if you're using pounds you might even get away with integers.

**Comment by WhyAsh5114** [comment]
Agreed tbh, `Float` is good enough. Avoids unnecessary complexity for Decimal types as well.
